<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemeWidget</class>
 <widget class="QWidget" name="ThemeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>22</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="platformStyleComboBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="iconThemeComboBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="colorSchemeComboBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
